nixos/modules/config/users-groups.nix

```diff
@@ -960,11 +966,14 @@ in {
           (filter (x: x != null) (map (flip getAttr user) passwordOptions));
       in optional (!unambiguousPasswordConfiguration) ''
         The user '${user.name}' has multiple of the options
-        `hashedPassword`, `password`, `hashedPasswordFile`, `initialPassword`
-        & `initialHashedPassword` set to a non-null value.
+        `initialHashedPassword`, `hashedPassword`, `initialPassword`, `password`,
```

Comment:
The trailing comma should go here: there's an `&` the next char.

Reply:
TIL